modules/tinymce/src/themes/silver/test/ts/module/TestExtras.ts

```diff
@@ -77,6 +77,7 @@ export const TestExtras = (): TestExtras => {
     off: editorOff,
     insertContent: (_content: string, _args?: any) => {},
     execCommand: (_cmd: string, _ui?: boolean, _value?: any) => {},
+    getContainer: () => SugarBody.body().dom,
```

Comment:
Is this function even used?

Reply:
yes, it is needed because otherwise some tests in `modules/tinymce/src/themes/silver/test/ts/headless/window/WindowManagerTest.ts` fail

Reply:
Right it's a mock thought it was a utility function didn't look a the whole file. So never mind. :+1: